.meshrc.yaml

```diff
@@ -60,6 +60,12 @@ sources:
             responseTypeName: MetadataFields
             headers:
               Cookie: "{context.headers['cookie']}"
+          - type: Query
+            field: MetadataValues
```

Comment:
I thought it'd been mentioned we would name this field `SamplesMetadata`.  Was that not viable?

Reply:
I just changed it last minute willy nilly! Is `SamplesMetadata` better?